I tested this using `sudo nixos-rebuild test -I nixpkgs=<path/to/local/nixpkgs> --fast`. My local configuration.nix has `calibre` in `systemPackages`. 

I tried `nixpkgs-review pr 131477` as per the instructions but it errored with:
```
Traceback (most recent call last):
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/bin/.nixpkgs-review-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/__init__.py", line 10, in main
    cli.main(command, args)
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/cli/__init__.py", line 262, in main
    return cast(str, args.func(args))
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/cli/pr.py", line 65, in pr_command
    contexts.append((pr, builddir.path, review.build_pr(pr)))
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/review.py", line 222, in build_pr
    return self.build_commit(base_rev, pr_rev)
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/review.py", line 167, in build_commit
    merged_packages = list_packages(
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/review.py", line 330, in list_packages
    return parse_packages_xml(f)
  File "/nix/store/lrz7786xlj8vc9hkhf4v6w7d36hfrvk2-nixpkgs-review-2.6.1/lib/python3.9/site-packages/nixpkgs_review/review.py", line 297, in parse_packages_xml
    values = (e.attrib["value"] for e in elem.getchildren())
AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'
```

I *think* this is an issue with the nixpkgs-review script (or my python setup) rather than the changes in this PR, but I am not sure.

--- a/pkgs/applications/misc/calibre/default.nix
+++ b/pkgs/applications/misc/calibre/default.nix
@@ -1,6 +1,7 @@
 { lib
 , mkDerivation
 , fetchurl
+, fetchFromGitHub
 , poppler_utils
 , pkg-config
 , libpng
@@ -94,7 +95,15 @@ mkDerivation rec {
       python
       regex
       sip
-      zeroconf
+      (zeroconf.overrideAttrs (oldAttrs: rec {
+        version = "0.31.0";
+        src = fetchFromGitHub {
+          owner = "jstasiak";
+          repo = "python-zeroconf";
+          rev = version;
+          sha256 = "158dqay74zvnz6kmpvip4ml0kw59nf2aaajwgaamx0zc8ci1p5pj";
+        };
+      }))
       # the following are distributed with calibre, but we use upstream instead
       odfpy
     ] ++ lib.optional (unrarSupport) unrardll

